tools/macosx-firewall.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Script that adds rules to Mac OS X Socket Firewall to avoid
+# popups asking to accept incoming network connections when
+# running tests.
+SFW="/usr/libexec/ApplicationFirewall/socketfilterfw"
+TOOLSDIR="`dirname \"$0\"`"
+TOOLSDIR="`( cd \"$TOOLSDIR\" && pwd) `"
+ROOTDIR="`( cd \"$TOOLSDIR/..\" && pwd) `"
+OUTDIR="$TOOLSDIR/../out"
+OUTDIR="`( cd \"$OUTDIR\" && pwd) `"
```

Comment:
Can you add a comment explaining the use of pwd (what you wrote in your comment earlier)?